.travis.yml

```diff
@@ -22,33 +22,41 @@ before_script:
 
 jobs:
   include:
-    - stage: Verify
+    - stage: Build
       script:
         - make install.tools
         - make .gitvalidation
         - make verify
+        - make binaries
       go: 1.8.x
     - script:
         - make install.tools
         - make .gitvalidation
         - make verify
+        - make binaries
       go: tip
-    - stage: Build and Test
+    - stage: Test
       script:
         - make install.deps
         - make test
-        - make binaries
```

Comment:
make test-cri will build anyway so removing make binaries just postpones the step